### PR TITLE
pages*: remove bold and italic styling

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
As described here: https://github.com/tldr-pages/tldr/pull/11768#discussion_r1428854895

### CODE REVIEW

File: pages/common/strings.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 `strings {{path/to/file}}`
 
-- Limit results to strings at least *length* characters long:
+- Limit results to strings at least n characters long:
 
 `strings -n {{length}} {{path/to/file}}`
```
Comment (gutjuri) [inline pages/common/strings.md:12]:
```suggestion
`strings -n {{n}} {{path/to/file}}`
```

The variable you refer to in the description is now called `n`?

File: pages.ca/linux/conky.md [hunk 2]
```diff
@@ -15,13 +15,13 @@
 
 `conky -c {{ruta/a/la/configuració}}`
 
-- Executa en segon pla (*daemon*):
+- Executa en segon pla (daemon):
```
Comment (kbdharun) [inline pages.ca/linux/conky.md:18]:
```suggestion
- Executa en segon pla (daemonize):
```

**Review by kbdharun — APPROVED** [review]
LGTM, thanks for the fixes.

**Review by gutjuri — APPROVED** [review]

File: pages.es/common/gradle.md
Comment (kant) [inline pages.es/common/gradle.md:1]: LGTM

File: pages.es/common/git-pull.md
Comment (kant) [inline pages.es/common/git-pull.md:1]: LGTM

File: pages.es/linux/conky.md
Comment (kant) [inline pages.es/linux/conky.md:1]: LGTM

**Comment by kbdharun** [comment]
This PR is GTG, the translations can be updated separately later.

**Comment by Magrid0** [comment]
>     * The page `pages.id/common/mv.md` is outdated, based on number of commands.

I misread 'id' as 'it' can someone check #11818 ?

**Comment by Magrid0** [comment]
>     * The page `pages.it/common/tac.md` needs a clean copy and translatio @Magrid0

done in #11819

**Pull request merged**